source/blog/y-combinator.md

```diff
@@ -4,6 +4,7 @@ excerpt: "Can you get into Y Combinator? Absolutely. Here's why and how."
 author_name: Chas Ballew
 author_email: chas@aptible.com
 author_id: chas
+hn_url: https://www.aptible.com/blog/y_combinator.html
```

Comment:
This is the one-off fix needed for this post, which falls in the unfortunate Venn Diagram overlap of: posts with a new link being redirected (from: `/blog/y_combinator.html` to: `/blog/y-combinator`) and posts with an existing HackerNews submission/discussion.